collections/binary_heap_test.ts

```diff
@@ -0,0 +1,269 @@
+import { assertEquals } from "../testing/asserts.ts";
+import { BinaryHeap } from "./binary_heap.ts";
+import { ascend, descend } from "./comparators.ts";
+
+export class MyMath {
+  multiply(a: number, b: number): number {
+    return a * b;
+  }
+}
+
+export interface Container {
```

Comment:
These two don't need to be `export`ed

Reply:
My bad. I originally had these in a different file called test_common.ts which I use in multiple test files of mine. I forgot to remove the export when I copied it into collections/binary_heap_test.ts. If I'm going to be using these objects again in my binary search tree and red black tree test coverage, should I inline them like I did here or create a test data folder for storing these classes used in multiple test cases?

Reply:
We loosely have the rule in deno_std that if the filename started with `_`, then that file is considered private. So maybe we can define `collections/_test_utils.ts` if we need some common items only used for testing.

Reply:
Done, I've moved those 2 classes to `collections/_test_utils.ts`.